OneSignalSDK/onesignal/src/main/java/com/onesignal/OSNotificationAction.java

```diff
@@ -57,4 +59,18 @@ public ActionType getType() {
    public String getActionID() {
       return actionID;
    }
+
+   public JSONObject toJSONObject() {
+      JSONObject mainObj = new JSONObject();
+
+      try {
+         mainObj.put("type", type.ordinal());
+         mainObj.put("actionID", actionID);
```

Comment:
`actionID` should be `actionId`

Reply:
should I change de variables too? and getter to getActionId?

Reply:
I think we should for consistency

Reply:
I agree for consistently as well